CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+- Update CreateOnly and ReadOnly javadocs to be more accurate that the validation is performed by RestLiValidationFilter.
```

Comment:
nit: format type names as code by wrapping in backticks (`)

Reply:
Updated